src/helpers/consts.js

```diff
@@ -34,6 +34,9 @@ export const CT_KITEI_MAKOR           = 'KITEI_MAKOR';
 export const CT_PUBLICATION           = 'PUBLICATION';
 export const CT_LELO_MIKUD            = 'LELO_MIKUD';
 
+// Source content type
+export const CT_SRC          = 'SOURCE';
```

Comment:
What is this ?
content type is for units and collections. not sources.

Reply:
It is used in canonicalLink in link.js:
 64   // source
 65   if (entity.content_type === CT_SRC){
 66     return `/sources/${entity.id}`;
 67   }

Want to call it differently?